.expeditor/verify.pipeline.yml

```diff
@@ -0,0 +1,33 @@
+steps:
+  - label: ":ruby: unit"
+    command:
+      - bundle install --binstubs --path=.vendor
+      - bundle exec rspec
+    expeditor:
+      executor:
+        docker:
+
+  - label: ":kubernetes: static"
```

Comment:
I think we can remove this test. It was specific to the old implementation.

Reply:
Done